SystemInfo/Collector/SymfonySystemInfoCollector.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+/**
+ * File containing the SymfonySystemInfoCollector class.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace EzSystems\EzSupportToolsBundle\SystemInfo\Collector;
+
+use EzSystems\EzSupportToolsBundle\SystemInfo\Value;
+use Symfony\Component\HttpKernel\Kernel;
+
+/**
+ * Collects information about the Symfony installation we are using.
+ */
+class SymfonySystemInfoCollector implements SystemInfoCollector
```

Comment:
Since we only collect kernel data, it would be `ConfigurationSymfonyKernelSystemInfoCollector` then..?